src/Uno.Extensions.Reactive.UI/AssemblyInfo.cs

```diff
@@ -1,3 +1,6 @@
 ﻿using System.Runtime.CompilerServices;
+using Uno.Extensions.Equality;
 
 [assembly: InternalsVisibleTo("Uno.Extensions.Reactive.Tests")]
```

Comment:
Can we move these properties into the csproj and then remove the AssemblyInfo.cs?

Reply:
I saw both pattern in the solution ... not sure which one I prefer. IMO the property in *.csproj is more for "generic" (like `$(AssemblyName).Tests`, but this does not apply here since we have only one tests project for all reactive packages for instance). And the advantage of the attribute / AssemblyInfo.cs file is to have a single place where we do have all those kind of configurations for the assembly (like the attribute I added).

BTW I also think that we should globally review usage of the `InternalsVisibleTo` in the solution and keep only those for tests projects.